### PR TITLE
library-manager 2.0.48 (new formula)

### PR DESCRIPTION
**Pull request by zengxs**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

[`Microsoft Library Manager`](https://github.com/aspnet/LibraryManager) is a web development tool for install and consume 3rd-party client-side libraries with ease.

### CODE REVIEW

File: Formula/library-manager.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+class LibraryManager < Formula
+  desc "Client-side web library manager"
+  homepage "https://github.com/aspnet/LibraryManager"
+  url "https://az320820.vo.msecnd.net/packages/microsoft.web.librarymanager.cli.2.0.48.nupkg"
+  sha256 "ca4f4ab4e57da62d650607b4a7a73b27a23dc2453896eda8d104ddc17e3b42a8"
+
+  resource "runtime" do
+    url "https://download.microsoft.com/download/9/1/7/917308D9-6C92-4DA5-B4B1-B4A19451E2D2/dotnet-runtime-2.1.0-osx-x64.tar.gz"
```
Comment (SMillerDev) [inline Formula/library-manager.rb]: This is not acceptable in core https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae. You can either make a formula out of it, or depend on the cask I guess.

File: Formula/library-manager.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  def install
+    system "unzip", "microsoft.web.librarymanager.cli.#{version}.nupkg"
+    (libexec/"libman").install Dir["tools/netcoreapp2.1/any/*"]
+
+    resource("runtime").stage do
+      (libexec/"dotnet-runtime").install Dir["*"]
+    end
+
+    (bin/"libman").write <<~EOS
```
Comment (SMillerDev) [inline Formula/library-manager.rb]: You probably want to look at `write_env_script` to make this cleaner

**Review by SMillerDev — COMMENTED** [review]
Thanks for the PR

**Comment by fxcoudert** [comment]
Closing due to shipping binaries, and lack of activity.

**Pull request closed without merging**